src/worker.ts

```diff
@@ -1,3 +1,14 @@
-export function init(): void {
-  console.log("Hello from Worker!"); // eslint-disable-line no-console
+import "unfetch/polyfill";
+import "native-promise-only";
+
+interface Response {
+  json(): Promise<Config>;
+}
+
+export function init(): Promise<Config> {
+  // FIXME; Use real API location, whatever that is
+  return fetch("https://api.fastly.com/api/url").then(
```

Comment:
I think we should move the URL string to a `src/constants.ts` module, as there will be a few more we'll need to hardcode somewhere and import.